server/instrumentation-backend/src/main/java/sh/calaba/instrumentationbackend/actions/location/FakeGPSLocation.java

```diff
@@ -142,16 +142,9 @@ private void setLocation(LocationManager locationManager, String locationProvide
             location.setLongitude(longitude);
             location.setAccuracy(1);
             location.setTime(System.currentTimeMillis());
-
-            try {
-                Method makeComplete = Location.class.getMethod("makeComplete");
-                if (makeComplete != null) {
-                    makeComplete.invoke(location);
-                }
-            } catch (Exception e) {
-                //Method only available in Jelly Bean
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+                location.setElapsedRealtimeNanos(SystemClock.elapsedRealtimeNanos());
```

Comment:
import android.os.SystemClock ?

Reply:
yes, I know. I had two checkouts and was compiling in the wrong one :-(